core/reflection.jvm/src/kotlin/reflect/jvm/internal/calls/CallerImpl.kt

```diff
@@ -131,6 +131,12 @@ internal sealed class CallerImpl<out M : Member>(
             }
         }
 
+        /**
+         * A default caller whose original caller is a instance method.
+         * see [KT-71378](https://youtrack.jetbrains.com/issue/KT-71378)
```

Comment:
I don't understand this comment. What's "original caller" and why is it an instance method? `copy` is a static method.

Reply:
`copy(Duration, Bool)` (`copy(long, boolean)` on jvm) is an instance method and the generated method `copy$default(Test, long, boolean, int, Object)` is static. 
We are creating default caller which calls `copy$default` so **original caller** here is the instance method `copy`. 
Necessary comments have been added